core/domain/feedback_jobs_one_off.py

```diff
@@ -100,3 +103,63 @@ def map(item):
     @staticmethod
     def reduce(key, value):
         pass
+
+
+class SuggestionMigrationOneOffJob(jobs.BaseMapReduceOneOffJobManager):
+    """One-off job for converting all suggestions from the old model to the
+    new model.
+    """
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [feedback_models.SuggestionModel]
+
+    @staticmethod
+    def map(suggestion):
+        suggestion_id = (
```

Comment:
perhaps call this new_suggestion_id to easily differentiate it from suggestion.id?

Reply:
Done